composer.json

```diff
@@ -1,10 +1,15 @@
 {
     "name": "root/public_html",
+    "minimum-stability": "dev",
     "authors": [
         {
             "name": "Jovany",
             "email": "jovany@ua.fm"
         }
     ],
-    "require": {}
+    "require": {
+    "php": ">=5.4.0",
+        "willotter/funny-jokes": "^0.2.1",
```

Comment:
I fix it at this commit: https://github.com/2jovany/hw2/commit/6060ac045794308b84d13d7c7ba81007cee31bf7